src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -64,7 +70,14 @@ extends AbstractMap[Key, Value]
   private[this] val actualInitialSize = OpenHashMap.nextPositivePowerOfTwo(initialSize)
 
   private var mask = actualInitialSize - 1
-  private var table : Array[Entry] = new Array[Entry](actualInitialSize)
+
+  /** The hash table.
+    * 
+    * The table's entries are initialized to `null`, indication of an empty slot.
+    * A slot is either deleted or occupied if and only if the entry is non-`null`.
```

Comment:
> it's not clear to me whether this class intends to support null keys

the situation for `null` seems to be not clearly specified. for example, in https://issues.scala-lang.org/browse/SI-2935, martin says:

> we can't give a clear guideline. I would rather leave the spec of maps underspecified. Some maps will accept nulls but you should not rely on it for all maps

since `OpenHashMap` currently does support `null` keys, we cannot just change this behavior. is there a good reason to change it?